conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/tezos/TezosDataOperations.scala

```diff
@@ -128,6 +129,9 @@ class TezosDataOperations extends ApiDataOperations {
     * @return the most recent block, if one exists in the database.
     */
   def latestBlockIO()(implicit ec: ExecutionContext): DBIO[Option[Tables.BlocksRow]] =
+  // Couldn't this query be done in single database request?
+  // I believe we could simply sort + take. Wouldn't this be faster?
+  // Probably we should test this out first
```

Comment:
it depends on wether the db-driver actually limits the results or prefetches them nonetheless, returning only the first

Reply:
Hm... followup question would be if that makes any difference for us either way. Meaning, if db-driver does (or not) that, wouldn't this be faster no matter what? We can talk about that here, if you like, but I'm going to remove this comment and leave it as it is. We could probably in the future take some time to do db optimizations.